.travis.yml

```diff
@@ -1,8 +1,41 @@
+---
 language: node_js
 node_js:
-  - "0.10.33"
-services: mongodb
+  - '0.10'
+
+sudo: required
+
+branches:
+  only:
+    - master
+    - develop
+
+git:
+  depth: 10
+
+services:
+  - mongodb
+
+cache:
+  directories:
+    - node_modules
+
 before_install:
-  - "npm install -g npm"
-  - "npm cache clean"
-  - "npm install -g adapt-cli"
+  - npm install -g npm
+  - npm install -g adapt-cli
+
+install:
+  - npm config set spin false
+  - npm install
+
+matrix:
+  fast_finish: true
+
+notifications:
+  email: false
```

Comment:
Definitely and those conversation email are generated by GitHub itself. As of now we haven't set any recipient for which Travis should dispatch email notification!

Please refer following docs - http://docs.travis-ci.com/user/notifications/#Email-notifications

This have clearly mentioned that we need to supply email id(s) of recipients who all are interested in receiving email when there is success and/or failure for PR(s).